### PR TITLE
CMR-7208: As a CMR client user, I would like to see API doc on granule bulk update.

### PR DESCRIPTION
**Pull request by ygliuvt**

### CODE REVIEW

File: ingest-app/docs/api.md [hunk 1]
```diff
@@ -1080,21 +1080,109 @@ Bulk update status and results are available for 90 days.
 
 ## <a name="granule-bulk-update"></a> Granule Bulk Update
 
-The granule bulk update API is used perform the same granule update to multiple concepts in one call.
+The granule bulk update API is used perform the same granule update to multiple granule concepts in one call.
 
-Bulk update is initiated through an ingest POST endpoint with the concept ids to update, the update type, the update field, and update information.
+Granule bulk update is initiated through an ingest POST endpoint with the bulk update operation, the update field, and the updates which is a list of granule urs and update values in the request body. See the [Granule Bulk Update JSON Schema](https://github.com/nasa/Common-Metadata-Repository/blob/master/ingest-app/resources/granule_bulk_update_schema.json) for the detailed format of granule bulk update request.
```
Comment (mschmele) [inline ingest-app/docs/api.md:1085]:
```suggestion
Granule bulk update is initiated through a POST to the ingest endpoint with the bulk update operation, the update field, and the updates, which is a list of granule urs and update values in the request body. See the [Granule Bulk Update JSON Schema](https://github.com/nasa/Common-Metadata-Repository/blob/master/ingest-app/resources/granule_bulk_update_schema.json) for the detailed format of granule bulk update request.
```

**Review by mschmele — APPROVED** [review]
One little phrasing change - looks good to me though!

File: ingest-app/docs/api.md [hunk 2]
```diff
@@ -1080,21 +1080,109 @@ Bulk update status and results are available for 90 days.
 
 ## <a name="granule-bulk-update"></a> Granule Bulk Update
 
-The granule bulk update API is used perform the same granule update to multiple concepts in one call.
+The granule bulk update API is used perform the same granule update to multiple granule concepts in one call.
 
-Bulk update is initiated through an ingest POST endpoint with the concept ids to update, the update type, the update field, and update information.
+Granule bulk update is initiated through a POST to the ingest endpoint with the bulk update operation, the update field, and the updates, which is a list of granule urs and update values in the request body. See the [Granule Bulk Update JSON Schema](https://github.com/nasa/Common-Metadata-Repository/blob/master/ingest-app/resources/granule_bulk_update_schema.json) for the detailed format of granule bulk update request.
```
Comment (mitchstartzel) [inline ingest-app/docs/api.md:1085]: Nit - `granule urs` would be more readable as `granule URs`

**Review by mitchstartzel — APPROVED** [review]

File: ingest-app/docs/api.md (hunk 2 above)
Comment (mitchstartzel) [reply ingest-app/docs/api.md:1085]: Looks good otherwise. Much needed documentation update

**Pull request merged**